web/src/components/MediaLibrary/MediaLibrary.tsx

```diff
@@ -0,0 +1,67 @@
+import { FieldError } from '@redwoodjs/forms'
+
+import Button from 'src/components/Button/Button'
+
+declare const cloudinary: any
+
+export interface ICloudinaryMediaLibraryAsset {
+  public_id: string
+  resource_type: string
+  type: string
+  format: string
+  version: number
+  url: string
+  secure_url: string
+  width: number
+  height: number
+  bytes: number
+  duration: string
+  tags: string[]
+  metadata: []
+  created_at: Date
+  access_mode: string
+  access_control: string[]
+  created_by: string
+  uploaded_by: string
+}
+
+interface IMediaLibraryProps {
+  name: string
+  handleMediaLibrary: (value: ICloudinaryMediaLibraryAsset[]) => void
+}
+
+const MediaLibrary = ({ name, handleMediaLibrary }: IMediaLibraryProps) => {
+  const onClickOpen = () => {
+    window.ml = cloudinary.openMediaLibrary(
+      {
+        cloud_name: 'dl5elpdjy',
+        api_key: '929416121543777',
+        username: 'nreliasar@gmail.com',
```

Comment:
Can we put this information in the `.env` file instead of exposing in to the interwebz, or doesn't it matter?

Reply:
Yes, that would be great. I cannot find how to use it from the `.env` file. Do you know how to do that?